docs/quickstart/index.md

```diff
@@ -8,40 +8,25 @@ sidenav: doc-side-quickstart-nav.html
 customjs: //survey.g.doubleclick.net/async_survey?site=dgxvheagfp4ai24o6y2ammm5fe
 ---
 <p class="lead">
-All you need to get started with gRPC
+Get started with gRPC
 </p>
 
 <div id="toc" class="toc mobile-toc"></div>
 
-These pages show you how to get up and running as quickly as possible in gRPC, including installing all the tools you’ll need and completing some basic tasks to run and modify a simple Hello World application. There’s a Quick Start for each gRPC supported language with accompanying sample code for you to explore and update. For a more detailed look at the same application and how it all works, see [Hello gRPC!] in our Guides section. For more tutorials and examples, see our [Tutorials]. You can read more about gRPC in general in [What is gRPC?]. 
+These pages show you how to get up and running as quickly as possible in gRPC, including installing all the tools you’ll need. 
```

Comment:
Can we put the sentence you removed back in? It actually puts the Quickstarts in context. Though I've just realized that we're missing Hello gRPC! in its entirety - it's basically the other half of the original Overview.  I'll do a separate PR with it tomorrow.